internal/criocli/criocli.go

```diff
@@ -1033,3 +1042,69 @@ func StringSliceTrySplit(ctx *cli.Context, name string) []string {
 
 	return values
 }
+
+func splitMapOptions(v string) (key string, values []string, err error) {
```

Comment:
nit, can this function be defined below the function that calls it so it can be read top to bottom